effortless-chef-io/effortless-config.md

```diff
@@ -134,55 +150,80 @@ This pattern is used to build policyfiles using the chef_repo pattern for organi
    cd $pkg_prefix
    hab pkg exec $pkg_origin/$pkg_name chef-client -z -c $pkg_prefix/config/bootstrap-config.rb
    ```
-1. Run Test Kitchen to ensure your cookbook works on Linux
-   ```
+
+1. Run Test Kitchen to ensure the cookbook works
+
+   Use this command to spin up a CentOS 7 virtual machine (VM) locally and run the cookbook using the latest Chef Infra Client:
+
+   ```bash
    kitchen converge base-centos
    ```
-   > Note: This will spin up a CentOS 7 VM locally and run your cookbook using the latest Chef Client. If you get errors in the Chef run you may need to supply attributes to your policyfile or make modifications so that your cookbook can run using the latest Chef Client
-1. When you are ready destroy the VM by running `kitchen destroy`
-1. You can now upload your policyfile pkg to builder by running the following
+
+   If you experience errors in this Chef run, you may need to supply attributes or make modifications to your Policyfile, so that your cookbook can run using the latest Chef Infra Client.
+
+1. When ready, delete the VM instance by running:
+
+   ```bash
+   kitchen destroy
    ```
+
+1. Upload the Policyfile pkg to Chef Habitat builder by running the following commands:
+
+   ```bash
    source results/lastbuild.env
    hab pkg upload results/$pkg_artifact
    ```
-1. To run your policyfile on a system you just need to install habitat as a service and run `hab svc load <your_origin>/<your_policyfile_name>`
+
+1. To run the Policyfile on a system, install Chef Habitat services and run:
+
+   ```bash
+   hab svc load <my_origin>/<my_policyfile_name>
+   ```
 
 ### Policyfile Cookbook pattern
 
-This is a pattern of building an artifact for a single cookbook. 
+This pattern builds an artifact for a single cookbook. View an example of this pattern [here](https://github.com/chef/effortless/tree/master/examples/effortless_config/policyfile_cookbook_pattern).
```

Comment:
```suggestion
The [cookbook pattern]((https://github.com/chef/effortless/tree/master/examples/effortless_config/policyfile_cookbook_pattern)) builds an artifact for a single cookbook.
```

Reply:
We need to be clear that we're providing an example Policyfile cookbook. Cookbooks vary.

Reply:
```suggestion
This pattern builds a Chef Habitat artifact for the Policyfile cookbook. You can find an [example Policyfile cookbook](https://github.com/chef/effortless/tree/master/examples/effortless_config/policyfile_cookbook_pattern) in the Chef Effortless GitHub repository.
```